### PR TITLE
Some stuffed ingredients

### PR DESCRIPTION
**Pull request by aleene**
**Description:**
Added some stuffed ingredients

**Related issues and discussion:** #4615

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -71174,10 +71174,17 @@ fr:Crème dessert saveur vanille
 fr:Crème dessert au chocolat
 # usage:fr:Crème dessert au chocolat: Lait partiellement écrémé 75%, sucre, amidon modifié de maïs, cacao maigre, chocolat 1,9% (pâte de cacao, sucre, émulsifiant : lécithine de tournesol), lactose** et deprotéines _lait,_ épaississants : carraghénanes - gomme xanthane.
 
+<en:pasta
+en:stuffed pasta
+fr:pâte farci, pâtes farcies
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:71179]:
```suggestion
fr:pâtes farcies
```

the singular does not really make sense here.

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -71223,6 +71230,8 @@ wikipedia:en:https://en.wikipedia.org/wiki/Ravioli
 # ingredient/ravioli has 92 products in 6 languages @2019-02-17
 # usage:fr:ravioli (water, durum wheat, 24% cream cheese, egg [open-air] 7%, mascarpone cheese, bread crumbs, coconut cream, 1% lemon juice, broth of vegetables, sugar, whey powder, juice of limes 0.3%, iodized cooking salt, lemon zest 0.2%, lemon extract 0.1%, syrup of limes, spices)
 
+<en:ravioli
+fr:ravioli farci aux legumes
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:71234]:
```suggestion
fr:raviolis farcis aux légumes
```

**Review by stephanegigandet — APPROVED** [review]

**Comment by VaiTon** [comment]
@aleene could you rebase pls?

**Comment by teolemon** [comment]
I've just rebased the branch @VaiTon @aleene

**Comment by teolemon** [comment]
Can you reapply @stephanegigandet 's 2 suggestions on top of it and then merge ?

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/ingredients.txt]: fixed in an upcoming commit

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/ingredients.txt]: fixed in an upcoming commit

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -71897,10 +71898,17 @@ fr:Crème dessert saveur vanille
...
+fr:pâtes farcies
+
+<en:stuffed pasta
+fr:pâtes fraiches farcies cuites
+
 # description:en:RAVIOLI are a type of dumpling comprising a filling enveloped in thin pasta dough.
 
 # <en:compound
-# <en:Stuffed Pastas
+<en:stuffed pasta, stuffed pastas
```
Comment (teolemon) [inline taxonomies/ingredients.txt:71911]:
```suggestion
<en:stuffed pasta
```

**Comment by teolemon** [comment]
merged, thanks @aleene and @stephanegigandet

**Pull request merged**